config.sample.yaml

```diff
@@ -18,6 +19,18 @@ data_source_connections:
 
 transform:
   spark_version: 3.3.1
+  datalake:
+    enabled: true
+    zones:  # Optional
```

Comment:
Just a few questions on naming: 
- Why are these things called "zones"?
- Why the zone names and container names are different? 
- Can we just call them "Raw", "Transformed", and "Curated" and drop the Bronze, Silver and Gold? From my experience, people sometimes mean different things when using metal names, and it could be confusing

Reply:
Zones are often used - https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/cloud-scale-analytics/best-practices/data-lake-zones

A zone can have multiple containers, for example it might have containers for different data sources, or data format.

Re naming - I always thought the same, but Bronze, Silver, Gold, seems to have been adopted more widely, as Raw, Curated and Enriched are too "fixed" and often need to add additional areas, hence these are kept to containers. this was just a stab at this, so might be wider discussion is required.